tests/collections/test_quality_matrix.py

```diff
@@ -433,6 +433,222 @@ def test_replication_source_quality_matrix():
+                        level=0,
+                    ),
+                    counts={
+                        "cover": 0,
+                        "short_title": 0,
+                        "title": 0,
+                        "description": 0,
+                        "status": 0,
+                        "url": 0,
+                        "language": 0,
+                        "learning_resource_type": 0,
+                        "mimetype": 0,
+                        "file_type": 0,
+                        "edu_context": 0,
+                        "age_range": 0,
+                        "fsk": 0,
+                        "taxon_id": 0,
+                        "classification": 0,
+                        "general_keywords": 0,
+                        "intended_end_user_role": 0,
+                        "curriculum": 0,
+                        "learning_time": 0,
+                        "duration": 0,
+                        "language_target": 0,
+                        "competence": 0,
+                        "competence_requirements": 0,
+                        "competence_check": 0,
+                        "oeh_quality_criminal_law": 0,
+                        "oeh_quality_copyright_law": 0,
+                        "oeh_quality_protection_of_minors": 0,
+                        "oeh_quality_personal_law": 0,
+                        "oeh_quality_data_privacy": 0,
+                        "oeh_quality_correctness": 0,
+                        "oeh_quality_currentness": 0,
+                        "oeh_quality_neutralness": 0,
+                        "oeh_quality_language": 0,
+                        "oeh_quality_medial": 0,
+                        "oeh_quality_didactics": 0,
+                        "oeh_quality_transparentness": 0,
+                        "oeh_accessibility_open": 0,
+                        "oeh_accessibility_find": 0,
+                        "accessibilitySummary": 0,
+                        "oeh_usability": 0,
+                        "oeh_interoperability": 0,
+                        "price": 0,
+                        "oeh_quality_login": 0,
+                        "oeh_accessibility_security": 0,
+                        "license_to": 0,
+                        "license": 0,
+                        "author_freetext": 0,
+                        "editorial_publisher": 0,
+                        "published_date": 0,
+                        "publishing": 0,
+                        "created": 0,
+                        "modified": 0,
+                        "versionLabel": 0,
+                        "publisher": 0,
+                        "node_uuid": 0,
+                        "published_handle_id": 0,
+                        "oeh_signatures": 0,
+                        "feedback_comment": 0,
+                        "creator": 0,
+                        "provider": 0,
+                        "validator": 0,
+                    },
+                    total=0,
```

Comment:
these rows were ommited before, because their count was zero and hence they did not show up in the elastic search response.

Reply:
In principle, we could make the `counts` dictionary optional, and only provide it if `total > 0`, as it does not provide any extra information otherwise.